test/controllers/update/sign.out.controller.spec.ts

```diff
@@ -36,7 +36,7 @@ describe("SIGN OUT controller", () => {
     jest.clearAllMocks();
   });
   describe("GET tests", () => {
-    test(`renders the ${config.SIGN_OUT_PAGE} page, when FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME is active,  with ${config.SECURE_UPDATE_FILTER_PAGE} as back link`, async () => {
+    test(`renders the ${config.SIGN_OUT_PAGE} page, with ${config.SECURE_UPDATE_FILTER_PAGE} as back link`, async () => {
```

Comment:
i think this is the wrong test file being updated. Doesn't look like we have changed controllers/update/update.sign.out.controller.ts